core/tracepoint/defined_class_spec.rb

```diff
@@ -0,0 +1,17 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'TracePoint#defined_class' do
+    class A; def foo; end; end
+
+    it 'returns class or module of the method being called' do
```

Comment:
Missing a spec showing for a Module. Also, what happens if a call is made within an inheritance chain (say because `super` is called a couple of times)

Reply:
yeah, I have added the specs & updated the PR.